$MirOS: ports/print/cups/patches/patch-test_run-stp-tests_sh,v 1.2 2006/06/21 19:25:36 bsiegert Exp $
--- test/run-stp-tests.sh.orig	Mon Jul 14 20:29:35 2008
+++ test/run-stp-tests.sh	Tue Oct 21 22:56:23 2008
@@ -21,7 +21,7 @@ argcount=$#
 # Make the IPP test program...
 #
 
-make
+${MAKE_PROGRAM}
 
 #
 # Figure out the proper echo options...
